packages/terra-site/src/examples/date-picker/DatePickerOnChange.jsx

```diff
@@ -1,15 +1,30 @@
 import React from 'react';
 import DatePicker from 'terra-date-picker';
 
-const handleOnChange = (event, selectedDate) => {
-  window.console.log('**handleOnChange** The selected date is: ', selectedDate);
-};
-
-const DatePickerOnChange = () => (
-  <DatePicker
-    name="date-input-onchange"
-    onChange={handleOnChange}
-  />
-);
+class DatePickerOnChange extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { date: '' };
+    this.handleDateChange = this.handleDateChange.bind(this);
+  }
+
+  handleDateChange(event, date) {
+    this.setState({ date });
+  }
+
+  render() {
+    return (
+      <div>
+        <div id="selected-index">
```

Comment:
Is there a need for the extra div wrappers? I maybe missing it, but I don't see the selected-index id used in other places.